platform/graphics/platform_graphics.c

```diff
@@ -136,21 +137,20 @@ void plDeleteFrameBuffer(PLFrameBuffer *buffer) {
         return;
     }
 
-#if 0
-    glDeleteFramebuffers(1, &buffer->fbo);
-    glDeleteRenderbuffers(1, &buffer->rbo);
-#endif
+    CallGfxFunction(DeleteFrameBuffer, buffer);
+
+    //TODO: Dealloc object here
 }
 
-void plBindFrameBuffer(PLFrameBuffer *buffer) {
-    if(buffer == NULL) {
-        // todo, warning regarding invalid buffer blah blah
-        return;
-    }
+void plBindFrameBuffer(PLFrameBuffer *buffer, PLFBOTarget targetBinding) {
```

Comment:
For public API these should be consistent, so use https://en.wikipedia.org/wiki/Snake_case for any public variable names.